Jenkinsfile

```diff
@@ -1,6 +1,6 @@
 #!groovy
 node('vagrant') {
-
+timestamps{
```

Comment:
What does this timestamps do here?

Reply:
displays timestamps inside the build logs.